README.md

```diff
@@ -61,23 +59,35 @@ In particular, you can find examples of:
+but in some cases these functions as well as `Model` and `Msg` types tend to
+grow pretty quickly in an unmanageable way so you need to start breaking things
+down.
 
 There are [many ways](https://www.reddit.com/r/elm/comments/5jd2xn/how_to_structure_elm_with_multiple_models/dbkpgbd/)
-you can go about it. In particular the rest of this document will focus just on [separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).
-This technique is useful for isolating parts that really don't need to know too much about each other.
-It reduces scope of things particular module can operate with so the number of things programmer has to reason about while adding or changing behaviour is lower.
-In TEA this is especially touching `Msg` and `Model` types and `update` functions for managing the state.
-
-**It's important to understand that `init`, `update`, `view` and `subscriptions` are all isolated functions connected via `Browser.element`.
-In pure functional programming we're "never" really managing state ourselves but are rather composing functions that takes state as data and produce new version of it (`update` function in TEA).**
-
-Now let's have a look at how we can use `Cmd.map`, `Sub.map` and `Html.map` for concern separation in Elm application.
-We will nest `init`, `update`, `subscriptions` and `view` one into another and `map` them from child's to parent's types.
-Parent module is then using these units to manage just a subset of its overall state (`Model`).
-Here is how `Model` and `Msg` types of a parent application might look like:
+you can go about it. In particular the rest of this document will focus just on
+[separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns).
+This technique is useful for isolating parts that really don't need to know too
+much about each other.
+It reduces scope of things a particular module can operate with so the number of
+things the programmer has to reason about while adding or changing behaviour is
+lower. In TEA this is especially touching the `Msg` and `Model` types and `update`
+functions for managing the state.
+
+**It's important to understand that `init`, `update`, `view` and `subscriptions`
+are all isolated functions connected via `Browser.element`.
+In pure functional programming we're "never" really managing state ourselves but
+are rather composing functions that takes state as data and produce new version
+of it (`update` function in TEA).**
+
+Now let's have a look at how we can use `Cmd.map`, `Sub.map` and `Html.map` for
+concern separation in Elm application. We will nest `init`, `update`,
+`subscriptions` and `view` one into another and `map` them from child's to
+parent's types. Parent module is then using these units to manage just a subset
+of its overall state (`Model`). Here is how `Model` and `Msg` types of a parent
+application might look like:
```

Comment:
sorry, much of these are just justifying the docs to ~80 chars :sweat_smile: I guess I'll revert this to make it easier to find what actual changes in the text were made...

Reply:
I think it's better :+1: